recipe/bld.bat

```diff
@@ -2,15 +2,14 @@ set PATH=%PREFIX%\cmake-bin\bin;%PATH%
 
 mkdir build && cd build
 
-cmake -G "%CMAKE_GENERATOR%" ^
+cmake -GNinja ^
 	  -D CMAKE_BUILD_TYPE=Release ^
 	  -D BUILD_SHARED_LIBS=ON ^
 	  -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
 	  -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
 	  -D ENABLE_ZLIB_COMPRESSION=ON ^
-	  -D CMAKE_BUILD_TYPE=Release ^
```

Comment:
Do we still want this?

Reply:
This line was duplicated, it is still there.

Reply:
Ah right 😅 Thanks for pointing that out 😄